music21/test/musicXMLTest.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+import unittest
+from xml.etree import ElementTree as ET
+
+import music21 as m21
+
+DATA_FILE = Path(__file__).parent / 'data/Test1.mxl'
+
+class Test(unittest.TestCase):
+    def testDeserialization(self):
+        """Sample score deserializes properly"""
+        score = m21.converter.parse(str(DATA_FILE))
+        self.assertEqual(
+            [str(p.getInstrument()) for p in score.parts],
+            ['P1: Piano: Piano',
+             'P1: Piano: Piano',
+             'P2: Voice: Voice',
+             'P2: Voice: Voice']
+        )
+        self.assertTrue(all(len(p.getElementsByClass('Measure')) == 8 for p in score.parts))
```

Comment:
if this were to fail, IIRC the test runner would just show the truthiness of the expression, not the specific incorrect values. I suggest
`self.assertEqual([len(p.getElementsByClass('Measure')) for p in score.parts], [8, 8, 8, 8])`